packages/sources/mycryptoapi/src/endpoint/gasprice.ts

```diff
@@ -0,0 +1,34 @@
+import { Requester, Validator } from '@chainlink/ea-bootstrap'
+import { Config, ExecuteWithConfig, InputParameters } from '@chainlink/types'
+
+export const supportedEndpoints = ['gasprice']
+
+export interface ResponseSchema {
+  safeLow: number
+  standard: number
+  fast: number
+  fastest: number
+  blockNum: number
+}
+
+const customError = (data: any) => data.Response === 'Error'
+
+export const inputParameters: InputParameters = {
+  speed: false,
+}
+
+export const execute: ExecuteWithConfig<Config> = async (request, _, config) => {
+  const validator = new Validator(request, inputParameters)
+  if (validator.error) throw validator.error
+
+  const jobRunID = validator.validated.id
+  const speed = validator.validated.data.speed || 'standard'
```

Comment:
Looks like this doesn't match the README

```suggestion
  const speed = validator.validated.data.speed || 'fast'
```